tests/_support/UnitTester.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Inherited Methods.
+ *
+ * @method void wantToTest($text)
+ * @method void wantTo($text)
+ * @method void execute($callable)
+ * @method void expectTo($prediction)
+ * @method void expect($prediction)
+ * @method void amGoingTo($argumentation)
+ * @method void am($role)
+ * @method void lookForwardTo($achieveValue)
+ * @method void comment($description)
+ * @method void pause()
+ *
+ * @SuppressWarnings(PHPMD)
+ */
+class UnitTester extends Codeception\Actor
+{
+    use _generated\UnitTesterActions;
```

Comment:
We can remove this use statement.
This class doesn't exists.
if we need this class, we can add the class.